test/sdk.test.ts

```diff
@@ -1,45 +1,81 @@
+import { BrowserOptions } from '@sentry/browser';
+
 import { CapacitorOptions } from '../src';
-import { DEFAULT_OPTIONS_MOBILE, DEFAULT_OPTIONS_WEB, init } from '../src/sdk';
+import { init } from '../src/sdk';
 import { NATIVE } from '../src/wrapper';
 
 jest.mock('../src/wrapper', () => {
   return {
     NATIVE: {
       platform: 'web',
-      initNativeSdk: jest.fn(),
+      initNativeSdk: jest.fn(() => Promise.resolve()),
     },
   };
 });
 
-describe('SDK Init', () => {
-  it('Uses default web options on web', () => {
-    NATIVE.platform = 'web';
+beforeEach(() => {
+  jest.clearAllMocks();
+});
 
-    init({ dsn: '' }, (finalOptions: CapacitorOptions) => {
-      expect(finalOptions.enableNative).toBe(false);
-      expect(finalOptions).toEqual(
-        expect.objectContaining(DEFAULT_OPTIONS_WEB),
-      );
-    });
-  });
+describe('SDK Init', () => {
+  // [string, platform, options, autoSessionTracking, enableNative, enableAutoSessionTracking]
+  const table: Array<[
+    string,
+    string,
+    CapacitorOptions & BrowserOptions,
```

Comment:
Shouldn’t this be a union type?

Reply:
I don't think so, it should be this: https://github.com/getsentry/sentry-capacitor/blob/36fd6a585d2f6f391f937e44411c7cfb7c75a10c/src/sdk.ts#L22

Which should be an intersection of `CapacitorOptions` and the options for the sibling SDK